python/tests/expressions/test_expressions.py

```diff
@@ -202,48 +178,36 @@ def test_notnan_bind_nonfloat():
     assert NotNaN(Reference("i")).bind(schema) == AlwaysTrue()
 
 
-@pytest.mark.parametrize(
-    "op, string",
-    [
-        (And(ExpressionA(), ExpressionB()), "And(left=ExpressionA(), right=ExpressionB())"),
-        (Or(ExpressionA(), ExpressionB()), "Or(left=ExpressionA(), right=ExpressionB())"),
-        (Not(ExpressionA()), "Not(child=ExpressionA())"),
-    ],
-)
-def test_strs(op, string):
-    assert str(op) == string
-
-
 def test_ref_binding_case_sensitive(table_schema_simple: Schema):
-    ref = Reference[str]("foo")
+    ref = Reference("foo")
     bound = BoundReference[str](table_schema_simple.find_field(1), table_schema_simple.accessor_for_field(1))
     assert ref.bind(table_schema_simple, case_sensitive=True) == bound
 
 
 def test_ref_binding_case_sensitive_failure(table_schema_simple: Schema):
-    ref = Reference[str]("Foo")
+    ref = Reference("Foo")
     with pytest.raises(ValueError):
         ref.bind(table_schema_simple, case_sensitive=True)
 
 
 def test_ref_binding_case_insensitive(table_schema_simple: Schema):
-    ref = Reference[str]("Foo")
+    ref = Reference("Foo")
     bound = BoundReference[str](table_schema_simple.find_field(1), table_schema_simple.accessor_for_field(1))
     assert ref.bind(table_schema_simple, case_sensitive=False) == bound
 
 
 def test_ref_binding_case_insensitive_failure(table_schema_simple: Schema):
-    ref = Reference[str]("Foot")
+    ref = Reference("Foot")
     with pytest.raises(ValueError):
         ref.bind(table_schema_simple, case_sensitive=False)
 
 
 def test_in_to_eq():
-    assert In(Reference("x"), (literal(34.56),)) == EqualTo(Reference("x"), literal(34.56))
+    assert In(Reference("x"), (34.56,)) == EqualTo(Reference("x"), 34.56)
```

Comment:
It seems weird to me that we are mixing `Reference("x")` with `(34.56,)`. Should we be consistent and either explicitly wrap literals or not wrap references?

Reply:
I see your point, my personal opinion, for the tests, I don't think it is an issue to mix them. I just remove the `Reference` here 👍🏻